Resources/migration.yml

```diff
@@ -53,7 +53,10 @@ FoodCondimentBottleSmallHotsauce: FoodCondimentBottleHotsauce
 FoodBakedCookieFortune: FoodSnackCookieFortune
 GunSafeSubMachineGunVector: GunSafeSubMachineGunDrozd
 
+# 2023-05-29
+OrGate: null
```

Comment:
why not migrate to a logic gate? or have prototypes based off the logic gate that default to certain types of gates

Reply:
because its not a logic gate, its 2 of them

Reply:
fair enough